autometa/taxonomy/ncbi.py

```diff
@@ -174,24 +179,23 @@ def get_lineage_dataframe(self, taxids, fillna=True):
             index = taxid
             columns = [superkingdom,phylum,class,order,family,genus,species]
 
-        .. NOTE::
-            If you would like to merge the returned DataFrame with another
-            DataFrame. Let's say where you retrieved your taxids:
-
-        Example 
+        Example
         -------
 
+        If you would like to merge the returned DataFrame ('this_df') with another
+        DataFrame ('your_df'). Let's say where you retrieved your taxids:
+
         .. code-block:: python
 
             merged_df = pd.merge(
-                your_df,
-                this_df,
+                left=your_df,
+                right=this_df,
                 how='left',
                 left_on=<taxid_column>,
                 right_index=True)
         """
         canonical_ranks = [r for r in reversed(NCBI.CANONICAL_RANKS)]
-        canonical_ranks.remove('root')
+        canonical_ranks.remove("root")
         taxids = list(set(taxids))
         ranked_taxids = {}
         for rank in canonical_ranks:
```

Comment:
In the following block, there are no checks that the taxid is valid or is in `names.dmp` or `merged.dmp` etc. (see other comments).

Reply:
Since this function is making use of `name()` function which is using `is_valid_taxid()` I thought it would be redundant to run it here again.
Let me know what you think about it

Reply:
Yeah, you are right. I guess I missed that part!